modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// */openshift_pipelines/installing-pipelines.adoc
+[id="op-installing-pipelines-operator-in-web-console_{context}"]
+= Installing the {pipelines-title} Operator in web console
+
+You can install {pipelines-title} using the Operator listed in the {product-title} OperatorHub. When you install the {pipelines-title} Operator, the custom resources required for the Pipelines configuration are automatically installed along with the Operator.
+
+[discrete]
+.Procedure
+
+. In the *Administrator* perspective of the web console, navigate to *Operators->OperatorHub*.
+
+. Use the *Filter by keyword* box to search for _{pipelines-title} Operator_ in the catalog. Click the *OpenShift Pipelines Operator provided by Red Hat* tile.
```

Comment:
I had deliberately added 'provided by Red Hat' to differentiate from community operator. However, in the sync call today I got to know the tile for both the operator ( community and productized) will be same( i.e will have *provided by Red Hat*), except there will be a label *community * added to the tile. 

Since both community operator and productized operator will be available in operator hub for sometime after TP. I will have to add a note asking user not to download community operator, but the productized one.